rke/k8s_rke_system_images.go

```diff
@@ -3649,7 +3649,7 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
 		// Reminder: This template contains ACI images which aren't in templates for v2.4.x
 		"v1.17.17-rancher2-2": {
-			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.3-rancher1",
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
```

Comment:
Are we updating etcd for v1.17.x? `We are consciously not updating k8s 1.17 as k8s is no longer supporting 1.17.`